iib/web/exception_handlers.py

```diff
@@ -0,0 +1,37 @@
+import functools
+
+import flask
+from kombu.exceptions import OperationalError
+
+from iib.exceptions import IIBError
+from iib.web import messaging
+
+
+def catch_rabbitmq_error(request):
```

Comment:
Could you please make the name more generic? The fact that IIB uses RabbitMQ as its broker is an implementation detail of the deployment and not the code.

A name such as `catch_broker_error` would likely be more appropriate.